charts/routes/templates/mlp-vs.yaml

```diff
@@ -34,6 +34,11 @@ spec:
     {{- end }}
     {{- end }}
     {{- if .Values.feast.enabled }}
+    {{- if .Values.extraMappings }}
```

Comment:
is the block in the right place? looks like it is inside the feast.enabled block

Reply:
oh! fixed

Reply:
The last `end` is still inside the feast block, can you check.

Reply:
:doh: in too much of a rush